atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/benchmarks/KvsGetRowsColumnRangeBenchmarks.java

```diff
@@ -40,53 +40,49 @@
 @State(Scope.Benchmark)
 public class KvsGetRowsColumnRangeBenchmarks {
 
+    public static final int EXPECTED_NUM_CELLS = WideRowsTable.NUM_ROWS * WideRowsTable.NUM_COLS_PER_ROW;
+    public static final List<byte[]> ROWS = IntStream
+            .rangeClosed(0, WideRowsTable.NUM_ROWS - 1)
+            .mapToObj(WideRowsTable::getRow)
+            .collect(Collectors.toList());
+    public static final int CORRECT_CELL_BATCH_HINT = 10000;
```

Comment:
nit: the distinction here is not correctness, but whether or not they are aligned with the block size